src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+class Node<T> {
+    private T data;
+    private Node<T> next;
+    private Node<T> prev;
+
+    Node(T data) {

```

Comment:
[When creating a new Node<T>, pass all the fields to the Node constructor.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#when-creating-a-new-nodet-pass-all-the-fields-to-the-node-constructor)